storage/src/mmr/bitmap.rs

```diff
@@ -61,7 +63,11 @@ impl<H: CHasher> Default for Bitmap<H> {
 }
 
 impl<H: CHasher> Bitmap<H> {
-    const CHUNK_SIZE: usize = H::Digest::SIZE;
+    /// The size of a chunk in bytes.
+    pub const CHUNK_SIZE: usize = H::Digest::SIZE;
```

Comment:
nit: I'm not sure if this is the right PR, but I suggest just making this a const or making `CHUNK_SIZE` a generic?